deploy/charts/openebs-monitoring/values.yaml

```diff
@@ -199,6 +199,8 @@ alertRules:
     enabled: true
   Jiva:
     enabled: true
+  lvm-LocalPV:
```

Comment:
Should be lvmLocalPV

Reply:
@rajaSahil pls confirm

Reply:
@avishnu This is the directory name where lvm based alerts are kept for helm charts. We can name it `{localpv flavour}-LocalPV`.

Reply:
Hmm, not sure on the name @rajaSahil . I remember we had decided during the mixin code review.

Reply:
Yes, that is the configuration for a mixin. But this is the directory name so should it be the same as `lvmLocalPV`?

Reply:
This seems a configuration to me and should match the one present in mixin ideally. If its a folder name as well, that's an implementation quirk.

Reply:
Ok. Then keeping the name as lvmLocalPV only.